src/Action/AbstractPostArchiveAction.php

```diff
@@ -16,12 +16,12 @@
 use Sonata\NewsBundle\Model\BlogInterface;
 use Sonata\NewsBundle\Model\PostManagerInterface;
 use Sonata\SeoBundle\Seo\SeoPageInterface;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Translation\TranslatorInterface;
 
-abstract class AbstractPostArchiveAction extends Controller
+abstract class AbstractPostArchiveAction extends AbstractController
```

Comment:
This is a BC break, but it should be okay. Can you please leave a note in the UPGRADE notes.

Reply:
We decide to not add support for `Symfony 5` in `Sonata 3`. It will be add in `Sonata 4`. It mean `Controller` should not be change to `AbstractController` in `3.x` branch but in `master`. Also it is `Action` so we should not extend them by `Controller`.